.travis.yml

```diff
@@ -19,5 +19,10 @@ cache:
     - ".eslintcache"
     - "node_modules"
 
+deploy:
+  provider: npm
+  email: "jared.farago@gmail.com"
+  api_key: "20189b3a-58c8-41fb-8c20-65864395a046"
```

Comment:
Is this ok to be public? You can use a public key with travis: https://docs.travis-ci.com/user/encryption-keys/

Reply:
id rather do that, i generated a new key on my npm account. i dont use it for anything else, but i would rather implement it the right way